cspell.words.txt

```diff
@@ -9,6 +9,9 @@ behat
 dtstamp
 langcode
 langcodes
+TZNAME
+TZOFFSETTO
+TZOFFSETFROM
```

Comment:
@ronaldtebrake @vcsvinicius Can we ensure the dictionary stays sorted alphabetically (so T after S)? :D Just like traditional print dictionaries, that makes words easier to find in the list :)